static/src/javascripts/projects/common/modules/ui/dropdowns.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import bean from 'bean';
+import fastdom from 'fastdom';
+
+const containerSelector = '.dropdown';
+const buttonCN = 'dropdown__button';
+const contentCN = 'dropdown__content';
+
+const updateAria = (container: Element): void => {
+    const v: boolean = container.classList.contains('dropdown--active');
+    const content = [...container.getElementsByClassName(contentCN)];
+    const button = [...container.getElementsByClassName(buttonCN)];
```

Comment:
so if `content = [a, b]` and `button = [c, d]` then `[...content, ...button] = [a, b, c, d]`

I'm not sure I understand the question 😅